webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -12,7 +12,7 @@ const locales = window.locales || emptyLocales;
 let locale = document.getElementsByTagName('html')[0].lang;
 
 locale = locale.length === 0 ? 'en' : locale.replace(/-/g, '_');
-export const jed = new Jed(locales[locale]);
+export const jed = new Jed(locales[locale] || locales['en']);
```

Comment:
does it make sense to log this to somewhere?

Reply:
updated to log failures to the console